### PR TITLE
Arn 1688 drug use yes route

### PR DESCRIPTION
**Pull request by Yasmin-jones**

### CODE REVIEW

File: app/sbna-poc/v1_0__initial-form/fields.ts [hunk 1]
```diff
@@ -10,6 +10,36 @@ const immigrationAccommodationHint = {
   `,
 }
 
+const impactOfDrugUsePhysicalHint = {
```
Comment (LucasCairns) [inline app/sbna-poc/v1_0__initial-form/fields.ts:13]: We could specify these directly on the field, for example

```javascript
{
  some_field: {
    // other properties..
    hint: { text: 'the hint text' }
  },
}
```

**Review by LucasCairns — DISMISSED** [review]
LGTM 👍

File: app/sbna-poc/v1_0__initial-form/fields.ts (hunk 1 above)
Comment (Yasmin-jones) [reply app/sbna-poc/v1_0__initial-form/fields.ts]: Done

**Review by LucasCairns — APPROVED** [review]
LGTM

**Pull request merged**